examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        segment = df[segment_name]
</pre><p>Suggest to slightly refactor this section for readability(don't make "timestamp" column as index above):</p><pre class="ql-syntax" spellcheck="false">segment_slice = pd.DataFrame({
  "timestamp": segment.index,
  "target": df[segment_name].values,
  "segment": [segment_name] * periods}
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1070/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>